x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -76,7 +76,8 @@ export const uiSettings: Record<string, UiSettings> = {
     }),
     value: true,
     description: i18n.translate('xpack.observability.enableComparisonByDefaultDescription', {
-      defaultMessage: 'Enable the comparison feature in APM app',
+      defaultMessage:
```

Comment:
Since the key has not been changed, the translation might not be updated in main ([discussion](https://elastic.slack.com/archives/C5TQ33ND8/p1688459235633549))

 Also, for the previous releases, since the translation is already done, there are basically two options:
- Removing the translation and, as a result, showing the English version by default
- Keep the current translation if it is OK to have different messages in different languages

Is the second option what you want?